### PR TITLE
New design for overview cards

### PR DESCRIPTION
**Pull request by derjubu**
Improve the design of the overview cards

### CODE REVIEW

File: src/app/components/CardTitle/CardTitle.tsx [hunk 1]
```diff
@@ -13,12 +13,9 @@ export default function CardTitle({ children }: CardTitleProps): JSX.Element {
...
   justify-self: center;
-  grid-column: 1/-1;
   width: 100%;
   margin: 0;
   font-size: 2rem;
-  color: var(--color-primary);
-  background-color: var(--color-background-primary);
   margin-bottom: 0.5rem;
-  padding: 0.5rem 0;
+  padding: 0.5rem 1rem;
```
Comment (janhendrikstruve) [inline src/app/components/CardTitle/CardTitle.tsx:20]: I saw that you're sometimes using px, sometimens rem, i you don't have a specific reason for that, i would maybe consider making it consistent :)

**Review by janhendrikstruve — APPROVED** [review]
Great code, nothing to complain about!

File: src/app/components/CardTitle/CardTitle.tsx (hunk 1 above)
Comment (SeanPed) [reply src/app/components/CardTitle/CardTitle.tsx:20]: wmcis

File: src/app/components/OverviewCard/OverviewCard.tsx [hunk 2]
```diff
@@ -37,21 +37,17 @@ export default function OverviewCard({
...
 const Card = styled.article`
-  max-width: 1400;
   margin-left: auto;
   margin-right: auto;
-  display: grid;
-  grid-template-columns: repeat(6, 1fr);
-  column-gap: 5px;
-  border: black solid 1px;
   border-radius: 5px;
+  max-width: 640px;
```
Comment (SeanPed) [inline src/app/components/OverviewCard/OverviewCard.tsx:43]: i like

**Review by SeanPed — COMMENTED** [review]
lgtm

**Pull request merged**